docs/ref/django-admin.txt

```diff
@@ -185,7 +185,7 @@ Example usage:
 
 .. django-admin-option:: --ignore PATTERN, -i PATTERN
 
-Ignores directories matching the given :mod:`glob`-style pattern. Use
+Ignores directories matching the given :py:mod:`glob`-style pattern. Use
```

Comment:
Hi @felixxm !
Sorry for the ping, but in the previous PR #17353 , you mentioned [the `:py:` directive link should be used with links pointing to the official documentation](https://github.com/django/django/pull/17353#discussion_r1374118675).

I understand that to mean adding  `:py:` to be standard modules as I've done here. Would that be correct? 
Thanks!

Reply:
References that you've changed works even without `:py:`. Please revert these changes.

I was only talking about using `:py:` for ` _import search path`.